actioncable/app/assets/javascripts/action_cable.coffee.erb

```diff
@@ -3,6 +3,7 @@
 
 @ActionCable =
   INTERNAL: <%= ActionCable::INTERNAL.to_json %>
+  PROTOCOLS: <%= ActionCable::PROTOCOLS.to_json %>
```

Comment:
How about adding `:protocols` to `INTERNAL` instead? It's a general purpose data bag.